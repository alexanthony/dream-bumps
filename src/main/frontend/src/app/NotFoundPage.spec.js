import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from './NotFoundPage'
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const component = shallow(
    <NotFoundPage />
  );
  expect(component).toBeDefined();
  expect(toJson(component)).toMatchSnapshot()
});
