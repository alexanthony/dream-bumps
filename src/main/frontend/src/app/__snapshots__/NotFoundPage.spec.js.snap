exports[`test Renders properly 1`] = `
<div>
  <h4>
    404 Page Not Found
  </h4>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/">
     Go back to homepage 
  </Link>
</div>
`;
